DSP_Plugins.GalacticScale/DSP_Plugins.GalacticScale.csproj

```diff
@@ -59,6 +59,9 @@
     <Reference Include="UnityEngine.CoreModule">
       <HintPath>..\Libs\UnityEngine.CoreModule.dll</HintPath>
     </Reference>
+    <Reference Include="UnityEngine.ImageConversionModule">
+      <HintPath>..\Libs\UnityEngine.ImageConversionModule.dll</HintPath>
```

Comment:
I think we can kick this dependency. I only added it when trying to export textures from code as PNG, but it shouldn't be used anywhere now. The texture editing is not reliant on format conversion.